src/plugins/data/public/ui/filter_bar/filter_item.tsx

```diff
@@ -65,26 +64,21 @@ class FilterItemUI extends Component<Props, State> {
   public render() {
     const { filter, id } = this.props;
     const { negate, disabled } = filter.meta;
-    let hasError: boolean = false;
+    let errorMessage: string | undefined;
 
     let valueLabel;
     try {
       valueLabel = getDisplayValueFromFilter(filter, this.props.indexPatterns);
     } catch (e) {
-      getNotifications().toasts.addError(e, {
-        title: this.props.intl.formatMessage({
-          id: 'data.filter.filterBar.labelErrorMessage',
-          defaultMessage: 'Failed to display filter',
-        }),
-      });
-      valueLabel = this.props.intl.formatMessage({
+      errorMessage = valueLabel = this.props.intl.formatMessage({
```

Comment:
Notification was removed as it renders multiple times - showing the error message on the filter is clear enough.